ios/RNSSearchBar.mm

```diff
@@ -348,4 +372,36 @@ - (UIView *)view
 RCT_EXPORT_VIEW_PROPERTY(onFocus, RCTBubblingEventBlock)
 RCT_EXPORT_VIEW_PROPERTY(onBlur, RCTBubblingEventBlock)
 
+RCT_EXPORT_METHOD(focus : (NSNumber *_Nonnull)reactTag)
```

Comment:
It seems risky to treat a native view as a module at the same time. In `react-native-svg`, we had such case and when migrating to new arch, we splitted it to a module and a view. Also, there are ways to not use the bridge in order to support the future bridgeless mode, and I think it would be nice if `react-native-screens` applied to it too (see e.g. https://github.com/software-mansion/react-native-svg/pull/1847/files#diff-92804e3153cd488997303254fe5c7e4ea48521712823779a79bca1edde23f899)

Reply:
On the other hand, I see you use commands, so why are those methods needed at all?